src/static/styles/App.scss

```diff
@@ -1,593 +1,642 @@
 @import "../../../node_modules/bootstrap/scss/bootstrap";
 @import "./ldbtn.scss";
 @import "./loading.scss";
-
 $primary-theme: #7B81FF;
 $secondary-theme: #FFBB7B;
 $secondary-dark-theme: #262867;
 $text-red: #F24D4D;
 $footer-color:#413f57;
 $footer-text-color: rgb(183, 183, 192);
 $footer-text-color-accent: rgb(230, 230, 230);
-
-$theme-colors: (
-"primary-theme": $primary-theme,
-"secondary-theme": $secondary-theme,
-"secondary-dark-theme": $secondary-dark-theme
-);
-
+$theme-colors: ( "primary-theme": $primary-theme, "secondary-theme": $secondary-theme, "secondary-dark-theme": $secondary-dark-theme);
```

Comment:
I noticed that this .scss file has a lot of formatting changes even though I think the content is mostly the same. Is there a reason for all the formatting changes (indents, moving stuff to one line)? If there isn't a particular reason, I think we should avoid unnecessary code changes.